### PR TITLE
text change

### PR DESCRIPTION
**Pull request by Richkum**
change some text to make it more attractive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h1>Hair STudio</h1>
         <section id="feature">
             <div class="title-text">
                 <p>FEATURES</p>
-                <h>Why Choose Us</h>
+                <h1>Why Choose Us</h1>
```
Comment (gastonche) [inline index.html]: We should have a question mark after the word

File: index.html [hunk 2]
```diff
@@ -83,7 +83,7 @@ <h1>Affordable Cost</h1>
         <section id="service">   
             <div class="title-text">
                 <p>SERVICES</p>
-                <h>We Provide Better</h>
+                <h1>We Provide Better</h1>
```
Comment (gastonche) [inline index.html]: Better what?

**Review by gastonche — CHANGES_REQUESTED** [review]
Left a few comments. Please check it out and request another review.

File: index.html (hunk 2 above)
Comment (dorinengabdoh) [inline index.html]: I can't approve it

**Review by dorinengabdoh — CHANGES_REQUESTED** [review]
please check again  your code.

File: index.html [hunk 3]
```diff
@@ -40,7 +40,7 @@ <h1>Hair STudio</h1>
         <section id="feature">
             <div class="title-text">
                 <p>FEATURES</p>
-                <h1>Why Choose Us</h1>
+                <h1>Why Choose Us?</h1>
```
Comment (dorinengabdoh) [inline index.html:43]: good job

**Review by dorinengabdoh — APPROVED** [review]

**Pull request merged**